grakn-test/src/test/java/ai/grakn/test/graql/reasoner/Tests.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Grakn - A Distributed Semantic Database
+ * Copyright (C) 2016  Grakn Labs Limited
+ *
+ * Grakn is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Grakn is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Grakn. If not, see <http://www.gnu.org/licenses/gpl.txt>.
+ */
+
+package ai.grakn.test.graql.reasoner;
+
+import ai.grakn.graql.MatchQuery;
+import ai.grakn.graql.QueryBuilder;
+import ai.grakn.graql.internal.reasoner.query.QueryAnswers;
+import ai.grakn.test.GraphContext;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Test;
+
+
+import static ai.grakn.test.GraknTestEnv.usingTinker;
+import static org.junit.Assume.assumeTrue;
+
+public class Tests{
+
+    @ClassRule
+    public static final GraphContext testSet10 = GraphContext.preLoad("testSet10.gql");
+
+    @Before
+    public void onStartup() throws Exception {
+        assumeTrue(usingTinker());
+    }
+
+    @Test
+    public void testSet10() {
+        QueryBuilder iqb = testSet10.graph().graql().infer(true);
+        String queryString = "match (role1: $x, role2: $y) isa relation2;";
+        QueryAnswers answers = queryAnswers(iqb.parse(queryString));
+        Assert.assertTrue(!answers.isEmpty());
```

Comment:
Try this:
```
Assert.assertThat(answers, not(empty()));
```
This will report a readable error stating that it expected it to be "not empty".
`not` and `empty` are imported from `Matchers`.